extensions/default/src/getSopClassHandlerModule.js

```diff
@@ -86,6 +87,34 @@ function getSopClassUids(instances) {
   return sopClassUids;
 }
 
+/**
+ * Default handler for a instance list with an unsupported sopClassUID
+ */
+function getDisplaySetsFromUnsupportedSeries(instances) {
```

Comment:
I would refactor this into a separate file since it just adds noise to the important code